components/brave_rewards/browser/rewards_protocol_handler.h

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+
+#include "base/strings/strcat.h"
+#include "base/strings/string_util.h"
+#include "base/task/post_task.h"
+#include "brave/common/url_constants.h"
+#include "chrome/browser/external_protocol/external_protocol_handler.h"
+#include "chrome/browser/profiles/profile.h"
+#include "content/public/browser/browser_task_traits.h"
+#include "content/public/browser/browser_thread.h"
+#include "content/public/browser/render_process_host.h"
+#include "content/public/browser/render_view_host.h"
+#include "net/base/escape.h"
+#include "components/prefs/pref_service.h"
+#include "brave/components/brave_rewards/common/pref_names.h"
+
+namespace brave_rewards {
+
+bool IsRewardsEnabled(content::BrowserContext* browser_context) {
```

Comment:
Oh, also you shouldn't define methods in a header like this. Please move the definitions to .cc file along with any headers that aren't strictly required for the .h file

Reply:
done